sync-files/action.yaml

```diff
@@ -107,11 +118,19 @@ runs:
               if ! [ -f "$dest_path" ]; then
                 echo "Newly copy $source_path to $dest_path."
                 mkdir -p $(dirname "$dest_path")
-                cp "$source_file" "$dest_path"
+
+                eval "$pre_command" || true
```

Comment:
`|| true` is for this.
https://github.com/kenji-miyake/autoware-github-actions/blob/0d4a8c0d872e75eed1355386c13e6e3d6c266f35/.github/sync-files.yaml#L49

If there is no file yet, `{dest}` isn't found.
https://github.com/kenji-miyake/autoware-github-actions/runs/6427565108?check_suite_focus=true#step:3:1017